ow_plexil/src/plans/EuropaMission.plp

```diff
@@ -156,11 +156,11 @@ EuropaMission: Concurrence
       // destination.
 
       // Note that this node is checkpointed as a single step, because data
-      // needed to checkpoint the indivual steps (trench_x, trench_y,
+      // needed to checkpoint the individual steps (trench_x, trench_y,
       // ground_pos, parallel, and the GroundFound lookup) is not persistent.
       // In other words, there is no checkpointing within this node -- the
       // entire node would be repeated if a crash occurred during its execution.
-      // Jira OW-657 is for making this plan fully interruptable.
+      // Jira OW-657 is for making this plan fully interrutpable.
```

Comment:
You introduced a typo!  On the other hand, listing Jiras in open source is debatable.

Reply:
Oh no I did 🤣  My mistake